buildout.cfg

```diff
@@ -86,3 +86,5 @@ initialization +=
 
 [versions]
 sc.recipe.staticresources = 1.1b5
+setuptools = 42.0.2
+zc.buildout = 2.13.3
```

Comment:
Generally I like to pin the versions of the largest Plone supported on cfg of smallest version. So, when new versions are pinned in Plone, the build will fail, indicating that we have to update the versions. For exemple:

https://github.com/collective/collective.nitf/commit/10a3e7e340d4d4fe8863c505c104f139a9b048ea#diff-d3e088c4790fbe53fd3120d360cd08ddR3

Reply:
Interesting!
I have taken it over in this PR.